packages/ui/webpack.config.js

```diff
@@ -33,6 +40,7 @@ module.exports = (env, argv) => {
       ],
     }),
     new ForkTsCheckerWebpackPlugin(),
+    new webpack.DefinePlugin(envVariables),
```

Comment:
See here: [https://github.com/Joystream/pioneer/blob/7661cc48f5ee4afd1b200a2113e77121393c486c/packages/ui/webpack.config.js#L23-L26](https://github.com/Joystream/pioneer/blob/7661cc48f5ee4afd1b200a2113e77121393c486c/packages/ui/webpack.config.js#L23-L26)

We already have webpack define plugins as they acted as a feature flag in the past. Shouldn't this be merged together?

Reply:
definitely - done :)